digitaltwins/samples/service/javascript/get_digital_twin.js

```diff
@@ -19,11 +19,11 @@ async function main() {
   const credentials = new IoTHubTokenCredentials(process.env.IOTHUB_CONNECTION_STRING);
   const digitalTwinServiceClient = new DigitalTwinServiceClient(credentials);
 
-  // Get digital twin
+  // Get digital twin and retrieve the modelId from it
   const digitalTwin = await digitalTwinServiceClient.getDigitalTwin(deviceId);
 
-  // Print digital twin
   console.log(inspect(digitalTwin));
+  console.log("Model Id: " + inspect(digitalTwin.$metadata.$model))
```

Comment:
>) [](start = 66, length = 1)

OMG  Look at this.  You let a semi-colon get away.  It's probably going to hide under your desk and reproduce.  It's very expensive at that point.